### PR TITLE
#333 Add external member

### PR DESCRIPTION
**Pull request by stephanevieira75**
# Preview
https://user-images.githubusercontent.com/36481167/115032234-6250a700-9ec9-11eb-9ebd-62a35adf56d4.mp4

### CODE REVIEW

File: twake/backend/node/src/services/channels/services/channel/guests/service.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import { ChannelExecutionContext } from "../../../types";
+import ChannelGuestListener from "./listener";
+import { getLogger } from "../../../../../core/platform/framework";
+import UserServiceAPI from "../../../../user/api";
+import {
+  ChannelGuestPrimaryKey,
+  ChannelPendingEmails,
+} from "../../../../../services/channels/entities";
+
+const logger = getLogger("channel.default");
```
Comment (chamerling) [inline twake/backend/node/src/services/channels/services/channel/guests/service.ts:23]: `channel.guests`

**Review by chamerling — COMMENTED** [review]
WIll be happy to see a video of this :)

File: twake/backend/node/src/services/channels/services/channel/guests/service.ts (hunk 1 above)
Comment (stephanevieira75) [reply twake/backend/node/src/services/channels/services/channel/guests/service.ts]: Updated

**Comment by stephanevieira75** [comment]
> WIll be happy to see a video of this :)

There isn't that much I can show you for the moment but i will soon! 👍🏻

**Comment by chamerling** [comment]
1 The message "Search between guests and pending email" is strange

Capture d’écran 2021-04-08 à 11 55 36

2 "Remove from channel" is not the right sentence when the mail has been sent but the user is not in the channel yet

Capture d’écran 2021-04-08 à 11 56 00

3 When you add the user, the list is empty just after you click on "add", I think that you should display the email you just added

Capture d’écran 2021-04-08 à 11 56 37

Capture d’écran 2021-04-08 à 11 56 41

**Pull request merged**